### PR TITLE
(PC-14258) feat(ProfileDeletion): Add new DeleteProfileSuccess screen with account reactivation

### PR DESCRIPTION
**Pull request by aliraiki**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-14258

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [x] Added a **screenshot** for UI tickets.
- [x] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|   Simulator Screen Shot - iPhone 13 - 2022-04-28 at 10 12 11  |         |                 | Capture d’écran 2022-04-28 à 10 12 25              |                 |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/profile/pages/DeleteProfile/__tests__/DeleteProfileSuccess.test.tsx [hunk 1]
```diff
@@ -26,4 +36,15 @@ describe('DeleteProfileSuccess component', () => {
       )
     })
   })
+
```
Comment (WendyPassculture) [inline src/features/profile/pages/DeleteProfile/__tests__/DeleteProfileSuccess.test.tsx:39]: Il manque un test pour le bouton de retour vers la home

File: src/features/profile/pages/DeleteProfile/DeleteProfileSuccessV2.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      buttons={[
+        <TouchableLink
+          key={1}
+          as={ButtonPrimaryWhite}
+          wording={t`Retourner à l'accueil`}
+          navigateTo={navigateToHomeConfig}
+        />,
+        <ButtonTertiaryWhite key={2} wording={t`Réactiver mon compte`} icon={Again} />,
+      ]}>
+      <StyledBody>{t`😢 On est super triste de te voir partir.`}</StyledBody>
```
Comment (Lucasbeneston) [inline src/features/profile/pages/DeleteProfile/DeleteProfileSuccessV2.tsx:28]: L'emoji est une demande des UXs ? Parce que je pensais qu'on n'en mettait

**Review by Lucasbeneston — APPROVED** [review]
👍

File: src/features/profile/pages/DeleteProfile/DeleteProfileSuccessV2.tsx (hunk 2 above)
Comment (aliraiki) [reply src/features/profile/pages/DeleteProfile/DeleteProfileSuccessV2.tsx:28]: Oui ils veulent commencer à en mettre apparemment

**Pull request merged**